src/components/page/HeaderBreadcrumb/HeaderBreadcrumb.tsx

```diff
@@ -40,7 +41,9 @@ class HeaderBreadcrumb extends React.Component<BreadCrumbProps, {}> {
 
     return (
       <div>
-        <Breadcrumb className="reveal-breadcrumb">
+        <Breadcrumb
+          className={`reveal-breadcrumb${isPlanAssignmentPage ? ' plans-breadcrumb' : ''}`}
```

Comment:
What if instead of changing `Breadcrumb` we wrap it in some div in the assignment page (where the bg needs to be different), and then use that div to write targeted CSS to change the background color?

Reply:
addressed https://github.com/onaio/reveal-frontend/pull/1194/commits/c0af93a306855cc6eedfaf09cb5e74b76f0c6da0